apps/web/src/components/Markdown/courses-markdown-body.tsx

```diff
@@ -50,7 +50,7 @@ const CoursesMarkdownBody = ({
           <h3 className="text-2xl font-medium">{children}</h3>
         ),
         p: ({ children }) => (
-          <p className="text-blue-1000 text-base tracking-wide md:text-justify">
+          <p className="text-blue-1000 text-base tracking-wide md:text-justify [&_code]:text-left [&_code]:whitespace-pre-wrap [&_code]:block [&_code]:w-fit">
```

Comment:
![image](https://github.com/user-attachments/assets/bba44678-715e-4a05-a022-83fabeec33bf)
Still not good, it shouldn't be on 2 lines, it's Inline Latex not block latex
-> just use **has-[>code]:text-left** instead of **[&_code]:text-left [&_code]:whitespace-pre-wrap [&_code]:block [&_code]:w-fit**

Reply:
Done @kevinmulier